assets/js/blocks/product-search/block.tsx

```diff
@@ -35,7 +45,11 @@ const ProductSearchBlock = ( {
 
 	return (
 		<div className={ classes }>
-			<form role="search" method="get" action={ HOME_URL }>
+			<form
+				role="search"
+				method="get"
+				action={ HOME_URL as string | undefined }
```

Comment:
What are your thoughts on keeping `action={ HOME_URL }` and adding the type definition directly to https://github.com/woocommerce/woocommerce-blocks/blob/e70f41972acf247c8a3a354eb3a60eadbc93e311/assets/js/settings/shared/default-constants.ts#L13

In case we use `HOME_URL` again in the future, then it's already typed.

Reply:
Yes, that'll also resolve the issue and have an added benefit of reusability ...
Shall I push this change in this PR only or in next PR which i'll raise for subsequent issues..?

Reply:
Sure not a problem! 😃